### PR TITLE
Add support for PoolTogether V3

### PR DESCRIPTION
**Pull request by WyseNynja**
This still needs testing, and I think we need to change another place to add the protocol, but its a start.

I pulled the addresses from Etherscan. They aren't very consistent with their token names/symbols, but I mostly just used what they called them.

### CODE REVIEW

**Comment by WyseNynja** [comment]
#1120

File: rotkehlchen/chain/ethereum/defi/zerionsdk.py [hunk 1]
```diff
@@ -158,6 +158,56 @@ def _handle_pooltogether(normalized_balance: FVal, token_name: str) -> Optional[
     return None
 
 
+def _handle_pooltogether_v3(normalized_balance: FVal, token_name: str) -> Optional[DefiBalance]:
+    """Special handling for pooltogether v3"""
+    if 'COMP' in token_name:
+        comp_price = Inquirer.find_usd_price(A_COMP)
+        return DefiBalance(
+            token_address=to_checksum_address('0x27B85f596feB14e4B5fAA9671720a556a7608C69'),
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/defi/zerionsdk.py:166]: here and in the other places you would need to rebase on develop and use `string_to_ethereum_address` instead

**Review by LefterisJP — COMMENTED** [review]
Hey @WyseNynja thanks for the PR!

Have you tested this? Can you also give us an address (does not need to be yours) which would hold those tokens so that we can test and verify if works.

Finally I see linting tests failed. so that would need to be fixed.

**Comment by LefterisJP** [comment]
Closing this since the code has changed too much since this was written and should end up using the decoders.

**Pull request closed without merging**